Documentation/csi_block_storage_kc_pdfs.md

```diff
@@ -0,0 +1,42 @@
+# Publications and related information
+
+The following Portable Document Format \(PDF\) publications and related online information sources are available for the IBM® block storage CSI driver.
+
+To view a PDF file, you need Adobe™ Reader. You can download it at no charge from the [Adobe website](http://get.adobe.com/reader/).
+
+## Publications
+
+|IBM block storage CSI driver publication|Description|PDF download link|
+|----------------------------------------|-----------|-----------------|
+|*Release notes*|This publication describes requirements, compatibility, change log, and known issues information for version 1.6.0.<br />First Edition \(June 2021\) for version 1.6.0.|![PDF icon](PDF_icon.jpg) [Download](PDFs/IBM_block_storage_CSI_driver_1.6.0_RN.pdf)|
+|*User guide*|This publication describes how to prepare for, install, configure, and use IBM block storage CSI driver.<br />Publication number: SC27-9590-07.|![PDF icon](PDF_icon.jpg)[Download](PDFs/IBM_block_storage_CSI_driver_1.6.0_UG.pdf)|
+
+{% reusables.doc-resources %}
+
+## Related information and publications
+
+For more information about storage systems and orchestration platforms that are supported by IBM® block storage CSI driver, see the following resources.
+
+IBM resources
+
+-   [IBM SAN Volume Controller documentation](https://www.ibm.com/docs/en/sanvolumecontroller) \(ibm.com/docs/en/sanvolumecontroller)
```

Comment:
is there a reason to always include this long (scheme-less) url text?
I thought the point of a hyperlink is to hide those details

Reply:
the end part is for the PDF content. let's leave it in for now